packages/react-components/source/react/library/icon/icons.js

```diff
@@ -1029,7 +1029,20 @@ const icons = {
       />
     ),
   },
-
+  'plus-circle-filled': {
+    medium: (
+      <>
+        <path d="M7.75 15.9575C12.0302 15.9575 15.5 12.4877 15.5 8.20752C15.5 3.92731 12.0302 0.45752 7.75 0.45752C3.46979 0.45752 0 3.92731 0 8.20752C0 12.4877 3.46979 15.9575 7.75 15.9575Z" />
+        <path
+          fillRule="evenodd"
+          clipRule="evenodd"
```

Comment:
Per "Adding icons to the Design System" at https://puppet.style/#/React%20Components/Icon `clipRule` can be safely removed.